app/assets/stylesheets/fe_modal.scss

```
@@ -11,12 +11,25 @@
   }
 }
 
+.modal-content.small {
```

aprovecha las herramientas de SASS para no tener que andar repitiendo codigo

```suggestion
.modal-content {  
  @include tablet {
     &.narrow {...}
     &.medium {...}
     &.wide {...}
     &.extra-wide {...}
  }
}
```